app/src/main/java/ar/com/wolox/android/example/model/News.java

```diff
@@ -0,0 +1,38 @@
+package ar.com.wolox.android.example.model;
```

Comment:
Why Java?

Reply:
This class continue being on Java please change it to Kotlin